Description: Upstream changes introduced in version 0.9-beta2-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 abntex (0.9-beta2-1) unstable; urgency=low
 .
   * Initial release
 .
 The person named in the Author field signed this changelog entry.
Author: Marcos Talau <talau@users.sourceforge.net>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- abntex-0.9-beta2.orig/Makefile
+++ abntex-0.9-beta2/Makefile
@@ -11,11 +11,6 @@ export TEXDIR= ${shell kpsewhich --expan
 
 export USRLOCAL=/usr/local/bin
 
-#Do "man lyx":
-#The system directory is determined by  searching  for  the
-#       file  "chkconfig.ltx".
-export LYXDIR=${dir ${shell  find /usr/share/lyx -name chkconfig.ltx}}
-
 #diretório corrente onde se expandiu o cvs do abntex
 export ABNTEXDIR=$(notdir $(shell pwd))
 
@@ -119,12 +114,11 @@ linux-doc-tgz: doc
 
 #install e uninstall específico para tetex localizado em ${TEXDIR}
 install-tex:
-	cp ${OPVR} --preserve texmf/* ${TEXDIR}
-
-install-lyx:
-ifneq (${LYXDIR},)
-	cp ${OPVR} --preserve lyx/* ${LYXDIR}
-endif
+	echo ${OPVR} # cp --verbose --recursive
+	echo ${TEXDIR}
+	echo ${DESTDIR}
+	mkdir -p ${DESTDIR}${TEXDIR}
+	cp ${OPVR} --preserve texmf/* ${DESTDIR}${TEXDIR}
 
 install-bin:
 	cp ${OPVR} --preserve bin/* ${USRLOCAL}
@@ -132,7 +126,7 @@ install-bin:
 post-install:
 	texhash
 
-install: install-tex install-bin install-lyx
+install: install-tex
 
 #instalação mínima a partir de um diretório cvs/abntex útil para testes
 #não inclui documentação
